editor/plugin.js

```diff
@@ -8,8 +8,9 @@
  org.ekstep.questionunitMTF.EditorPlugin = org.ekstep.contenteditor.questionUnitPlugin.extend({
    initialize: function() {
      this._super();
-     var templatePath = ecEditor.resolvePluginResource(this.manifest.id, this.manifest.ver, 'editor/templates/mtf-template.html');
-     var controllerPath = ecEditor.resolvePluginResource(this.manifest.id, this.manifest.ver, 'editor/controllers/mtf-controller.js');
+     var instance = this;
+     var templatePath = ecEditor.resolvePluginResource(instance.manifest.id, instance.manifest.ver, 'editor/templates/mtf-template.html');
```

Comment:
This has to handle by parent class. It has to read the manifest and load the template & controller when we call this.super() from derived class.